tests/modules/cluster_file/main.nf.test

```diff
@@ -0,0 +1,29 @@
+nextflow_process {
+    name "Test Process CLUSTER_FILE"
+    script "modules/local/cluster_file/main.nf"
+    process "CLUSTER_FILE"
+
+    test("Test when sample levels are different") {
```

Comment:
Thanks so much 😄 . Can you add in a success test case as well? Or is this something provided in another PR?

Reply:
Added here: [6691dea](https://github.com/phac-nml/gasnomenclature/pull/11/commits/6691dea10c7140403b859fb97ea9a2c4cc79149b)